prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -323,7 +323,9 @@ class FHIRConverter(
                         } else {
                             // We know from the null check above that this cannot be null
                             val bundle = processedItem.bundle!!
-                            transformer?.process(bundle)
+
+                            // Tried this a few ways, reportEventService ends up as null/not working in FHIRTransformer
+                            transformer?.process(bundle, reportEventService)
```

Comment:
Currently the code in this PR does not send an azure event when it gets triggered. The main issue is that reportEventService isn't usable by the FHIRTransformer. Somehow it ends up null/uncallable.